mcp-servers/github/README.md

```diff
@@ -0,0 +1,30 @@
+# Deploy GitHub MCP Server on OpenShift Container Platform
+
+This document provides instructions for deploying the GitHub MCP server on the OpenShift Container Platform.
+
+## Official GitHub MCP Server Repository
+
+You can find the official GitHub MCP server repository here: [GitHub MCP Server](https://github.com/modelcontextprotocol/servers/tree/main/src/github)
+
```

Comment:
Before you can run the `3_simple_agent_with_githubmcp.py` the mcp server needs to be up and running. Can you add instructions for that here too?